custom/lib/CreateUserArgs.php

```diff
@@ -0,0 +1,9 @@
+<?php
+
+namespace Passage\Client;
+
+use OpenAPI\Client\Model\CreateUserRequest;
+
+class CreateUserArgs extends CreateUserRequest
```

Comment:
couldn't get `class_alias` to resolve the imports correctly, so just went with creating new classes. These are only used with the new function signatures. All of the old ones still use the same codegen'd classes.

Reply:
Why is this needed?

`use OpenAPI\Client\Model\CreateUserRequest as CreateUserArgs;` wasn't working in the Passage.php file?

Reply:
`class CreateUserArgs extends CreateUserRequest` creates a new class with the name, which is available to be imported/used by any dev external to this file

`use OpenAPI\Client\Model\CreateUserRequest as CreateUserArgs;` only allows the `CreateUserArgs` name to be used within the file the line is written in